frontend/packages/integration-tests-cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Is this unused? I'd rather just add real fixtures when we have the need rather than having an unused example.

Reply:
It's a placeholder for when we get there :-)